demisto_sdk/commands/upload/tests/uploader_test.py

```diff
@@ -350,14 +349,23 @@ def test_upload_pack(demisto_client_configure, mocker):
         - Ensure pack is uploaded successfully
         - Ensure status code is as expected
         - Ensure amount of messages is as expected
+        - Check that all expected content entities that appear in the pack are reported as uploaded.
     """
     mocker.patch("builtins.print")
     pack_path = f"{git_path()}/demisto_sdk/tests/test_files/Packs/DummyPack"
     uploader = Uploader(input=pack_path, insecure=False, verbose=False)
     mocker.patch.object(uploader, 'client')
     status_code = uploader.upload()
+    expected_entities = ['DummyIntegration.yml', 'integration-UploadTest.yml', 'DummyScriptUnified.yml',
+                         'script-DummyScript.yml', 'DummyPlaybook.yml', 'incidenttype-Hello_World_Alert.json',
+                         'incidentfield-Hello_World_ID.json', 'incidentfield-Hello_World_Type.json',
+                         'incidentfield-Hello_World_Status.json', 'classifier-aws_sns_test_classifier.json',
+                         'widget-ActiveIncidentsByRole.json', 'layout-details-test_bla-V2.json',
+                         'upload_test_dashboard.json']
     assert status_code == 0
-    assert len(print.call_args_list) == 19
+    assert len(print.call_args_list) == 21
```

Comment:
can we avoid this number? can we actually calculate it? it is unclear what is `21`

Reply:
I can remove it.

Reply:
im ok with having it, just would be better if we can sum up all entities and getting to the number rather than having it hard coded

Reply:
mmm I see what you are saying but I am not sure it is the best thing here since some of the prints here relate to the unification process and stuff like this. so the sum of messages != sum of entities.

Reply:
ok so lets add comment explaining what that number is